### PR TITLE
Add new API convertToSinglePlacementGroup for VM Scale Set.

### PR DESCRIPTION
**Pull request by ZheWang711**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5760 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5760/commits/aa4c0fd3edecc1082ea6e6e01440e26319c5994a

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Comment by anuchandy** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Comment by anuchandy** [comment]
@ravbhatnagar addition of API to existing api-version, please review

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 1]
```diff
@@ -8394,7 +8437,15 @@
         }
       },
       "description": "LogAnalytics output properties"
-    }
+    },
+    "VMScaleSetConvertToSinglePlacementGroupInput": {
+      "properties":{
+        "activePlacementGroupId": {
+            "type": "string",
+            "description": "Id of the placement group in which you want future virtual machine instances to be placed. To query placement group Id, please use Virtual Machine Scale Set VMs - Get API. If not provided, the platform will choose one with maximum number of virtual machine instances."
```
Comment (anuchandy) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: this is a required property? if so please mark it as required

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 2]
```diff
@@ -3557,6 +3557,49 @@
...
+          {
+                "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        },
+        "x-ms-long-running-operation": false
```
Comment (anuchandy) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: we don't need to set `x-ms-long-running-operation` to false explicitly if an api is not LRO.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json (hunk 1 above)
Comment (ZheWang711) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: No, `activePlacementGroupId` is an optional parameter. If it's not present in customer's request, we'll internally select one as default if possible.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json (hunk 2 above)
Comment (ZheWang711) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Sure, I will remove this in next revision.

**Review by KrisBash — APPROVED** [review]
LGTM

**Review by anuchandy — APPROVED** [review]

**Pull request merged**